src/MenuButton/MenuButtonItem.js

```diff
@@ -16,6 +16,8 @@ MenuButtonItem.propTypes = {
   onSelect: PropTypes.func.isRequired,
   /** Optional start icon for menu item */
   startIcon: PropTypes.oneOf(VALID_ICON_NAMES),
+  /** Optional start icon for menu item */
```

Comment:
```suggestion
  /** Optional end icon for menu item */
```

Reply:
Thanks im closing this PR out in favor of https://github.com/narmi/design_system/pull/1529